spec/test-utils.js

```diff
@@ -12,10 +12,12 @@ const mockGithub = {
               title: 'My second fake issue'
             }]
           });
-        } else {
+        } else if (repo == 'nonexisting') {
```

Comment:
Yes. Any errors other than errors with status `Not Found` (repo does not exist) come under the `else` clause. This (`else if` clause) captures cases when the repo does not exist, but otherwise there are no issues.